spec/acceptance/sequence_resetting_spec.rb

```diff
@@ -1,4 +1,4 @@
-describe "FactoryBot.rewind_sequences" do
+describe "FactoryBot::Internal.rewind_sequences" do
```

Comment:
This should remain `FactoryBot.rewind_sequences` since we want to test the public method, not the internal one.